### PR TITLE
Requirements added

### LINKED ISSUE #3: Create requirements file
- Add a `requirements.txt` file containing only the required python packages to be installed through `pip`
 - This can be done by taking the output of `pip freeze` and keeping only the required packages.
 - This will be used for Travis CI
 - Refer [this link](https://pip.readthedocs.io/en/1.1/requirements.html) for more information on the requirements file.

### PR DESCRIPTION
**Pull request by dhruvhacks**
closes #3 

Affects-
* Includes a new file ```requirements.txt``` with required dependencies to run this project
* Adds Installation Instructions in ```README.md```

### CODE REVIEW

**Comment by dhruvhacks** [comment]
@Samyak2 Please review.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+matplotlib==3.0.3
+numpy==1.17.3
+pandas==0.25.2
+pyparsing==2.4.2
+python-dateutil==2.8.0
+pytz==2019.3
+scikit-learn==0.21.3
+scipy==1.3.1
+six==1.12.0
+sklearn==0.0
```
Comment (Samyak2) [inline requirements.txt]: Version 0.0?

Comment (dhruvhacks) [reply requirements.txt]: My bad. Did not notice shift from ```pip install sklearn``` to ```pip install scikit-learn```
Reference [link](https://pypi.org/project/sklearn/)

Comment (Samyak2) [reply requirements.txt]: Yeah

**Comment by dhruvhacks** [comment]
@Samyak2 Changes made. Please review!

**Comment by Samyak2** [comment]
Thank you for your contributions!

**Pull request merged**